lib/src/firebase_auth_mocks_base.dart

```diff
@@ -131,11 +131,15 @@ class MockFirebaseAuth implements FirebaseAuth {
       isEmailVerified: _verifyEmailAutomatically,
       displayName: 'Mock User',
       providerData: [
-        UserInfo({
-          'email': email,
-          'uid': id,
-          'providerId': 'password',
-        }),
+        UserInfo.fromPigeon(
+          PigeonUserInfo(
+            email: email,
+            uid: id,
+            providerId: 'password',
+            isAnonymous: false,
+            isEmailVerified: true
```

Comment:
Would you mind using `_verifyEmailAutomatically` instead of setting `isEmailVerified` to `true`?